Neptune.Tests/ServiceTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Newtonsoft.Json;
+using NSubstitute;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+//http://anthonygiretti.com/2018/09/06/how-to-unit-test-a-class-that-consumes-an-httpclient-with-ihttpclientfactory-in-asp-net-core/
```

Comment:
I'm gonna be honest. I think a lot of unit testing is a waste of time. E2E and integration are far more productive usually.

https://medium.com/@copyconstruct/testing-microservices-the-sane-way-9bb31d158c16

Reply:
I agree, but I think it be worth our while to unit test these shared libraries considering they will be used by all the other engineers. So at least we have SOME confidence as well as a built-in sample of how to use Neptune

Reply:
For experienced developers: Yes I agree. For students with little experience: I just need a place to point out what the type orchestration should look like, how to inject the clientFactory, and generally how to use the library. Unit tests are the easiest to get up and running.

I'm not genuinely trying to test the library, I'm trying to provide the smallest workable example of working code. The headless and very small nature of Unit tests are useful for this.